code/game/objects/items/changeling_vial.dm

```diff
@@ -0,0 +1,34 @@
+/obj/item/changeling_vial
+	name = "secure vial"
+	desc = "An old, dusty vial. The secure cap on the top doesn't look like it will be easy to remove."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "secure_vial"
+	flags = FPRINT
+	w_class = 1
+	unacidable = 1
+	var/genomes_to_give = 10 //seeing as the new changeling won't have had a whole round to prepare, they get some genomes free
+
+/obj/item/changeling_vial/attack_self(mob/user as mob)
+	if(istype(user, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = user
+		if(!(H.mind in ticker.mode.changelings))
+			user.visible_message("<span class='warning'>As [user] crushes the vial, a mass of black goo leaps at \his face!.</span>","<span class='warning'>As you try to remove the cap, you crush the vial in your hand!</span> <span class='danger'>A mass of black goo leaps at you from the vial!</span>")
+			playsound(get_turf(src), "shatter", 20, 1)
+			ticker.mode.changelings += H.mind
```

Comment:
Should I bother yet? This has no chance of getting merged until @Intigracy is done with antag datums, right?